### ISSUE TITLE
tridactyl causes the site help.sendowl.com to not load

### ISSUE THREAD

**Issue by nathanbraun** [tridactyl/tridactyl#3000]
Tridactyl version: 1.20.2pre4790-1f527f14
Firefox version: Mozilla Firefox 82.0.3
Operating system: mac

*   Brief description of the problem:
When I go to the site:
https://help.sendowl.com/help
with tridactyl running, I only see the top header.

*   Steps to reproduce:
Go to https://help.sendowl.com/help

*   URL of the website the bug happens on:
https://help.sendowl.com/help

*   Config (in a new tab, run `:viewconfig`, copy the url and paste it somewhere like gist.github.com):
https://gist.github.com/nathanbraun/fca278267d33836c650a98b914815077

**Comment by glacambre** [comment]
Have you tried the troubleshooting guide? https://github.com/tridactyl/tridactyl/blob/master/doc/troubleshooting.md#settings-that-can-fix-websites

**Comment by nathanbraun** [comment]
Yeah I did, one follow up — looks like the content loads, it just puts a huge space between the header and the rest of the content. So if you scroll down you can see what's normally there/what you'd see without Tridactyl.

It's not a big deal for me (esp now that I realized I can just scroll down), just wanted to report in case it helps make Tridactyl better. Thanks for all your work! Tridactyl is the reason I use FF.

**Comment by glacambre** [comment]
>just wanted to report in case it helps make Tridactyl better

Thank you for taking the time to do it :)

**Comment by Rummskartoffel** [comment]
I took a look at the site and found that there's a massive div at the top of not only the page, but the document, that seems to be creating that gap:
```html
<html>
  <div class="embed-responsive etc.">
    <iframe id="cmdline_iframe">
      <!-- no idea how Tridactyl's command line ended up here – or the div, for that matter -->
    </iframe>
  </div>
  <head>
  </head>
  <body>
  </body>
</html>
```
On my end, that div has a height of ~1070px, which is just over one page. It gets this height because of a `padding-bottom` of 56.25%, defined in an inline style sheet. Manually setting it to `unset` makes the site look normal. When I open the site in MS Edge – without Tridactyl, obviously – the div in question doesn't exist and the site looks normal.

**Comment by bovine3dom** [comment]
I've noticed massive white space at the top of a few pages (none of which I can remember the names of), perhaps it's a problem common to some framework?

**Comment by Rummskartoffel** [comment]
help.sendowl.com seems to be using Bootstrap 3.2.0, judging from a licence comment in one of the style sheets. I only know one other site that uses Bootstrap, https://speedrun.com, but that site works just fine for me.

**Comment by Rummskartoffel** [comment]
I decided to investigate again, and I found the culprit. The page's HTML contains this:
```html
<script>
  $(function() {
    $('iframe').wrap("<div class='embed-responsive embed-responsive-16by9'/>").addClass('embed-responsive-item');
  });  
</script>
```
I'm not sure how to best deal with this, if at all. An autocmd might do the trick.

**Comment by bovine3dom** [comment]
We could more generally check to see that the command line is where we left it a short time after inserting it into the page
